src/components/Board.js

```diff
@@ -10,24 +10,121 @@ import CARD_DATA from '../data/card-data.json';
 class Board extends Component {
   constructor() {
     super();
-
     this.state = {
       cards: [],
     };
   }
 
+  componentDidMount = () => {
+    let query =  this.props.url + this.props.boardName + '/cards'
+    console.log(query)
+
+    axios.get(query)
+      .then((response) => {
+        console.log(response);
+        this.setState({
+          cards: response.data
+        });
+      })
+      .catch((error) => {
+        this.setState({
+          error: error.message
+        });
+        if (error.message) {
+          this.renderError();
+          console.log(error.message);
+        }
+      }
+    );
+  }
+
+  deleteCard = (id, index) => {
+    let url = this.props.url + this.props.boardName + '/cards/' + id
+    console.log(id);
+    axios.delete(url)
+      .then((response) => {
+        console.log(response);
+        const updatedCards = this.state.cards
+        delete updatedCards[index]
```

Comment:
Just a note on how `delete` works.  It makes that element of the array `undefined` so if you did:

``
delete updateCards[3];
```
then the element at index 3 will become `undefined`.